pkgs/development/python-modules/plum-py/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, fetchFromGitLab, isPy3k, pytest, baseline }:
+
+buildPythonPackage rec {
+  pname = "plum-py";
+  version = "0.4.0";
+  disabled = !isPy3k;
+
+  src = fetchFromGitLab {
+    owner = "dangass";
+    repo = "plum";
+    rev = "6a9ff863c0e9fa21f7b2230d25402155a5522e4b";
```

Comment:
https://gitlab.com/dangass/plum/-/issues/128

Reply:
Thanks for filing that. My issue about the test failure is now two weeks old too. Should the lack of response hold up this PR?

Reply:
No, not really.